projects/js-packages/licensing/components/activation-screen/index.jsx

```diff
@@ -77,6 +78,16 @@ const ActivationScreen = props => {
 		}
 	}, [ availableLicenses ] );
 
+	const [ activePlugins, setActivePlugins ] = useState( [] );
```

Comment:
There's a whole load of state here. I think it might be worth us turning this into a little store with `useReducer` at some point, although I'm not sure if that's discouraged and we should use `@wordpress/data` instead.